BenchPress/Helpers/Azure/Bicep.psm1

```diff
@@ -1,3 +1,120 @@
+    0
+    Confirm-BicepFile: ../../../examples/actionGroupErrors.bicep:
+    Confirm-BicepFile: /workspaces/benchpress/examples/actionGroupErrors.bicep(6,7) : Warning no-unused-params: Parameter "location" is declared but never used. [https://aka.ms/bicep/linter/no-unused-params]
+    Confirm-BicepFile: /workspaces/benchpress/examples/actionGroupErrors.bicep(12,13) : Warning no-hardcoded-location: A resource location should not use a hard-coded string or variable value. Please use a parameter value, an expression, or the string 'global'. Found: 'westus' [https://aka.ms/bicep/linter/no-hardcoded-location]
+    1
+
+    BicepErrors
+    -----------
+    {@{Path=../../../examples/actionGroupErrors.bicep; ErrorResults=System.Collections.ObjectModel.Collection`1[System.Management.Automation.PSObject]…
+
+    ---------- Example 3: Provide -BicepPath Parameter ----------
+
+    Confirm-BicepFile -BicepPath ./examples/actionGroupErrors.bicep
+
+    Confirm-BicepFile: ../../../examples/actionGroupErrors.bicep:
+    Confirm-BicepFile: /workspaces/benchpress/examples/actionGroupErrors.bicep(6,7) : Warning no-unused-params: Parameter "location" is declared but never used. [https://aka.ms/bicep/linter/no-unused-params]
+    Confirm-BicepFile: /workspaces/benchpress/examples/actionGroupErrors.bicep(12,13) : Warning no-hardcoded-location: A resource location should not use a hard-coded string or variable value. Please use a parameter value, an expression, or the string 'global'. Found: 'westus' [https://aka.ms/bicep/linter/no-hardcoded-location]
+    0
+
+    BicepErrors
+    -----------
+    {@{Path=../../../examples/actionGroupErrors.bicep; ErrorResults=System.Collections.ObjectModel.Collection`1[System.Management.Automation.PSObject]…
+
+    ---------- Example 4: Path without -BicepPath Parameter ----------
+
+    Confirm-BicepFile ./examples/actionGroupErrors.bicep
+
+    Confirm-BicepFile: ../../../examples/actionGroupErrors.bicep:
+    Confirm-BicepFile: /workspaces/benchpress/examples/actionGroupErrors.bicep(6,7) : Warning no-unused-params: Parameter "location" is declared but never used. [https://aka.ms/bicep/linter/no-unused-params]
+    Confirm-BicepFile: /workspaces/benchpress/examples/actionGroupErrors.bicep(12,13) : Warning no-hardcoded-location: A resource location should not use a hard-coded string or variable value. Please use a parameter value, an expression, or the string 'global'. Found: 'westus' [https://aka.ms/bicep/linter/no-hardcoded-location]
+    0
+
+    BicepErrors
+    -----------
+    {@{Path=../../../examples/actionGroupErrors.bicep; ErrorResults=System.Collections.ObjectModel.Collection`1[System.Management.Automation.PSObject]…
+
+  .INPUTS
+    System.String[]
+
+  .OUTPUTS
+    System.Management.Automation.PSCustomObject[]
+#>
+function Confirm-BicepFile {
+  [cmdletbinding()]
+  [OutputType([System.Object[]])]
+  param(
+    [Parameter(Mandatory, Position=0, ValueFromPipeline)] [string[]]$BicepPath
```

Comment:
Consider renaming this parameter as `BicepFilePath`

Reply:
Updated per suggestion.